onecgiar-pr-server/src/api/results/result.repository.ts

```diff
@@ -754,7 +754,7 @@ WHERE
     }
   }
 
-  async getResultById(id: number): Promise<Result> {
+  async getResultById(id: number): Promise<any> {
```

Comment:
@JuanGuzman-io consider either typing the query response or using `unknown` instead

Reply:
@gcmartinezg by typing it as unknown causes conflict in the use of the properties